tests/assets/testing_state.cfg

```diff
@@ -3,10 +3,6 @@ foo=bar
 
 [state]
 jobid_01=J:123456
-mergerepo_01=git://git.kernel.org/pub/scm/linux/kernel/git/
-mergehead_01=master
-localpatch_01=/tmp/patch.txt
-patchwork_01=http://example.com/patch/1
```

Comment:
Maybe keep some of the data in the testing files and verify they haven't been changed in a test? We use a state file with a ton of data skt runner shouldn't be touching so this could be useful. Not sure though what your next steps are though so this might not be needed based on that.